guides/advanced_guides/field_properties.md

```diff
@@ -1,20 +1,54 @@
+
+In other words, the `searchableAttributes` list serves two purposes:
+
+1. It designates the fields that are searchable.
+2. It dictates the [attribute ranking order](/guides/main_concepts/relevancy.md#attribute-ranking-order).
+
+There are two possible modes for the `searchableAttributes` list.
+
+#### Default: Automatic
+
+**By default, all attributes are automatically added to the `searchableAttributes` list in their order of appearance.** This means that the initial order will be based on the order of attributes in the first document indexed, with each new attribute found in subsequent documents added at the end of this list.
+
+This default behavior is indicated by a `searchableAttributes` value of `["*"]`. To verify the current value of your `searchableAttributes` list, use the [get searchable attributes endpoint](/references/searchable_attributes.md#get-searchable-attributes).
+
+If you'd like to restore your searchable attributes list to this default behavior, simply [set `searchableAttributes` to an empty array `[]`](/references/searchable_attributes.md#update-searchable-attributes) or use the [reset searchable attributes endpoint](/references/searchable_attributes.md#reset-searchable-attributes).
 
-Their content is used by MeiliSearch to assess the relevancy of a document.
+#### Manual
 
-**By default, all field attributes are set as searchable**.
+You may want to make some attributes non-searchable, or change the [attribute ranking order](/guides/main_concepts/relevancy.md#attribute-ranking-order) after documents have been indexed. To do so, simply place the attributes in the desired order and send the updated list using the [update searchable attributes endpoint](/references/searchable_attributes.md#update-searchable-attributes).
 
-Therefore, if a field attribute is not in the searchable-attributes list, the field will be ignored from searches but will remain stored in the server. This list can be restricted to a selected set of attributes in the settings.
+::: warning
+Be aware that after manually updating the `searchableAttributes` list, subsequent new attributes will no longer be automatically added unless the settings are [reset](/references/searchable_attributes.md#reset-searchable-attributes).
+:::
 
 #### Example
 
-Suppose you manage a database that contains information about movies. By adding the following settings, the fields `uid`, `movie_id`, `title`, `description`, `poster`, `release_date` and `rank` will be searched.
+Suppose you manage a database that contains information about movies. By adding the following settings, the fields `uid`, `movie_id`, `title`, `description`, `poster`, `release_date` and `rank` will be searchable, and ranked in that order.
 
 <CodeSamples id="field_properties_guide_searchable_1" />
```

Comment:
Agree 100%!

Reply:
Since this code sample is precisely made for the guide that you have updated, could you create a new PR and change the `cURL` example in `.code-samples.meilisearch.yaml`. I will then see that there is a issue opened in every SDK directory to change their respective code-sample file.

Reply:
#679